src/main/java/uk/gov/pay/directdebit/app/config/DirectDebitModule.java

```diff
@@ -23,11 +47,26 @@ protected void configure() {
         bind(Environment.class).toInstance(environment);
     }
 
+    // Nasty hack alert - gocardless client does not seem to pick up the certificates in our trust store automatically, so we need to inject those. The underlying client (okhttp) supports that, but it's not accessible. So we make it accessible *** godmode ***. Sent an email to gocardless about the issue.
```

Comment:
Indeed. The proxy bit I've just added, and I need to get it merged to test it out and see if it makes a difference - otherwise I will remove it.
If we decide to keep it, I'll definitely add more context to the comment